src/JMS/Serializer/EventDispatcher/Subscriber/SymfonyValidatorSubscriber.php

```diff
@@ -21,7 +21,7 @@
 use JMS\Serializer\EventDispatcher\Event;
 use JMS\Serializer\EventDispatcher\EventSubscriberInterface;
 use JMS\Serializer\Exception\ValidationFailedException;
-use Symfony\Component\Validator\ValidatorInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
```

Comment:
@stof True, but 2.3 is EOM this month and EOL in a year. It seems that cutting a release of the Serializer library that supports 3.0 doesn't necessarily need to support 2.3. That being said, do you have a strategy / suggestion for how to fix this to work with both? Wrapping the use statement in a class_exists conditional or something, perhaps?